mapfishapp/src/test/java/org/georchestra/mapfishapp/ws/upload/UpLoadFileManagementGTImplTest.java

```diff
@@ -152,11 +154,11 @@ public void testKML22ExtendedData() throws Exception {
 		assertNotNull(getJsonFieldValue(properties, "f_annee"));
 	}
 
-    private String getJsonFieldValue(JSONObject properties, String field) {
+    private Object getJsonFieldValue(JSONObject properties, String field) {
 
-        String value;
+        Object value;
         try {
-            value = properties.getString(field);
+            value = properties.get(field);
```

Comment:
there is an optString method with return null, or a string, not sure wether it is available or not in the targetted version, in 20180813 for sure (as in master).